code/elemental/ElementalAreasPageExtension.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SilverStripe\FrameworkTest\Elemental\Extension;
+
+use DNADesign\Elemental\Extensions\ElementalAreasExtension;
+use DNADesign\Elemental\Models\ElementalArea;
+
+/**
+ * This is used to test multiple elemental areas on a page
+ */
+class ElementalAreasPageExtension extends ElementalAreasExtension
```

Comment:
Why not `extends ElementalPageExtension`?

It would satisfy me if we just say "this doesn't have to be for a page, it can be for any dataobject" and then just make this change:

```suggestion
 * This is used to test multiple elemental areas on a single data object
 */
class MultiElementalAreasExtension extends ElementalAreasExtension
```

Reply:
Rather confusingly, `ElementalPageExtension extends ElementalAreasExtension`

`ElementalAreasExtension` is what enables you to have on or more `ElementalArea`'s on `DataObject` and things get saved correctly

`ElementalPageExtension` is an essentially an implementation of that and it adds in single ElementalArea relation

However for the behat test I'm doing it's explicitly testing 2x ElementalArea's (testing more complexity that way) so I think it's much cleaner to extend the base `ElementalAreasExtension`

I'll update the naming